polyfills/Element/prototype/cloneNode/detect.js

```diff
@@ -1,6 +1,8 @@
 'document' in this && "cloneNode" in document.documentElement && (function() {
-	var test = document.createElement('input');
+	var div = document.createElement('div'), test = document.createElement('input');
 	test.checked = true;
+	div.appendChild(test);
 	var result = test.cloneNode();
-	return !!result.checked;
+	var result2 = div.cloneNode();
```

Comment:
Should I amend and force push or fix it is a separate commit? I don't have much experience with Github.